JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+  agent {
+    node {
+      label 'project:any'
+    }
+  }
+  stages {
+    stage('Clean Workspace') {
+      steps {
+        cleanWs()
+      }
+    }
+    stage('Git Clone') {
+      steps {
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']],
+        doGenerateSubmoduleConfigurations: false,
+        extensions: [],
```

Comment:
Are lines 17-18 needed?

Reply:
Probably not - cruft from current repository - will remove...